pkg/action/build-uki.go

```diff
@@ -321,9 +360,11 @@ func (b *BuildUKIAction) createInitramfs(sourceDir, artifactsTempDir string) err
 		if err != nil {
 			return fmt.Errorf("getting record of %q failed: %w", filePath, err)
 		}
+		if rec.Name != strings.TrimPrefix(rec.Name, sourceDir) {
+			rec.Name = strings.TrimPrefix(rec.Name, sourceDir)
+		}
```

Comment:
```suggestion

		rec.Name = strings.TrimPrefix(rec.Name, sourceDir)

```

Reply:
let me try this with and without to see if it works and search on why I added it :D

Reply:
yep, works. Not sure why I though it would not lol